tests/unit/backend/layers/business/test_business.py

```diff
@@ -1769,19 +1791,47 @@ def test_create_collection_version_with_migration_revision_flag_succeeds(self):
         # The new version should not be published (aka Private)
         self.assertIsNone(new_version.published_at)
 
-        # The new version has is_auto_version == True
+        # The canonical collection should be published
+        self.assertIsNotNone(new_version.canonical_collection.originally_published_at)
+
+        # get_collection still retrieves the original version
+        version = self.business_logic.get_published_collection_version(published_collection.collection_id)
+        self.assertEqual(version.version_id, published_collection.version_id)
+        self.assertNotEqual(version.version_id, new_version.version_id)
+
         self.assertTrue(new_version.is_auto_version)
 
-    def test_create_collection_version_with_migration_revision_flag_fails(self):
+    def test_create_collection_version_is_auto_version_succeeds__with_revision(self):
         """
-        If is_auto_version is True, a collection version can NOT be created if there is another
-        unpublished version with is_auto_version == True for that specific canonical collection
+        Given a canonical collection with a published version and an unpublished version where is_auto_version is False,
+        creating another unpublished collection version where is_auto_version is True will succeed.
         """
         published_collection = self.initialize_published_collection()
-        self.business_logic.create_collection_version(published_collection.collection_id, is_auto_version=True)
+        existing_revision = self.business_logic.create_collection_version(
+            published_collection.collection_id, is_auto_version=False
+        )
+        auto_revision = self.business_logic.create_collection_version(
+            published_collection.collection_id, is_auto_version=True
+        )
 
-        with self.assertRaises(CollectionVersionException):
-            self.business_logic.create_collection_version(published_collection.collection_id, is_auto_version=True)
+        # The new version has a different version_id
```

Comment:
nit: Use auto revision in the comments to keeps naming consistent. This is to avoid confusing it with the existing_revision.